README.md

```diff
@@ -20,12 +20,33 @@ terminus decoupled-kit:create
 Run with command line flags:
 
 ```
-terminus decoupled-kit:create my-site-name "My Site Label" --org="My Org" --cms=drupal --install-cms=FALSE
+terminus decoupled-kit:create my-site-name "My Site Label" drupal-composer-managed --org="My Org" --cms=drupal --install-cms=FALSE --region=us
 ```
 
-If you use the `--install-cms=FALSE` flag, the CMS sites won’t be installed automatically.
+Different Upstream options:
+
+Drupal 10:
```

Comment:
Explaining what the option does makes sense, but I don't think it makes sense to include a list of upstreams here.